content/docs/shared-content/need-help.md

```diff
@@ -3,4 +3,4 @@ updatedOn: '2023-11-24T11:25:06.767Z'
 ---
 ## Need help?
 
-Join the [Neon community forum](https://community.neon.tech/) to ask questions or see what others are doing with Neon. [Neon Pro Plan](/docs/introduction/pro-plan) users can open a support ticket from the console. For more detail, see [Getting Support](/docs/introduction/support).
+Join our [Discord Server](https://discord.com/invite/92vNTzKDGp) or [Discourse Community Forum](https://community.neon.tech/) to ask questions or see what others are doing with Neon. [Neon Pro Plan](/docs/introduction/pro-plan) users can open a support ticket from the console. For more detail, see [Getting Support](/docs/introduction/support).
```

Comment:
This could also be `https://neon.tech/discord/` since the `/discord` URL should always redirect to the correct invite link even if we change it.